docs/index.rst

```diff
@@ -0,0 +1,46 @@
+..
+    Copyright (C) 2018 CERN.
+
+    Asclepias Broker is free software; you can redistribute it and/or modify it
+    under the terms of the MIT License; see LICENSE file for more details.
+
+
+.. include:: ../README.rst
```

Comment:
I know this is still early days, but it would be good to have a description of the aims of the project and this package.

Reply:
Including the README is confusing because the README includes a link to this page, so it's a bit strange to have the self-reference.